File: src/main/java/practise/misc/FizzBuzz.java
package practise.misc;

public class FizzBuzz {

    public static void main(String[] args) {
        fizzBuzz();
    }

    private static void fizzBuzz() {
        for (int i = 1; i <= 100; i++) {
            String output = "";
            if (i % 3 == 0) output = "fizz";
            if (i % 5 == 0) output += "buzz";
            if (output.equals("")) output = String.valueOf(i);
            System.out.println(output);
        }
    }
}

File: src/main/java/practise/misc/FizzBuzz.kt
package practise.misc

fun main() {
    fizzBuzz()
}

private fun fizzBuzz() {
    for (i in 0..100) {
        when {
            i % 15 == 0 -> println("FizzBuzz")
            i % 5 == 0 -> println("Fizz")
            i % 3 == 0 -> println("Buzz")
            else -> println(i)
        }
    }
}